config/project-scratch-def.json

```diff
@@ -1,8 +1,9 @@
 {
-    "orgName": "wade.wegner Company",
+    "orgName": "GIFter",
     "edition": "Developer",
-    "orgPreferences" : {
-        "enabled": ["S1DesktopEnabled", "ChatterEnabled"],
-        "disabled": ["S1EncryptedStoragePref2"]
+    "settings" : {
```

Comment:
Hey @douglascayers - just curious why you aren't also "enabling" chatter and Lightning, per the original scratch def.

Reply:
Hi @ekapner, I left those out because they weren't necessary, at least not in my tests. New orgs come with Chatter and Lightning already enabled. It doesn't hurt to include them though to be overly explicit.